queries/fish/textobjects.scm

```diff
@@ -1,16 +1,66 @@
-(function_definition) @function.outer
+;; assignment
+(command
+  name: (word) @_command
+  argument: (word) @_varname @assignment.lhs @assignment.inner
+  argument: (_)* @assignment.rhs
+  (#not-lua-match? @_varname "[-].*")
+  (#eq? @_command "set")) @assignment.outer
 
+(command
+  name: (word) @_name
+  argument: (_)* @assignment.inner
+  (#eq? @_name "set"))
+
+; ;; block
+; ([
+;  (case_clause)
+;  (if_statement)
+;  (switch_statement)
+;  (else_clause)
+; ] (_) @block.inner) @block.outer
```

Comment:
Any reason this is commented out?

Reply:
Sorry, was because the `@block.inner` does not work yet. Forgot to remove it